### PR TITLE
First part of the housekeeping work around transfer

### PR DESCRIPTION
**Pull request by mszostok**
##### SUMMARY

This pull request is made against the `main` branch. This branch will become default after merging this PR.

- Remove license from file headers, 
- Remove `.mergify.yml`, 
- Update main license file,
- Change `develop` refs to `main`
- Change `ghcr.io` registry
- Remove the `$` from the copy-paste scripts. As a result, you can paste it to you terminal without any modification.
- Fix dead links: 
    ```bash
    npm install -g markdown-link-check
    # remote the CHANGELOG.md file first
    find . -name \*.md -print0 | xargs -0 -n1 markdown-link-check
    ```
- Update the `CHANGELOG.md` file.

Will be changed in the follow-up PR:
- `go.mod` and import statements
  - Go report badge
  - Go mod badge
- Helm releasing

#### After merging

- change the default branch to `main`
- delete `develop` branch
- ensure such settings:
    Screen Shot 2022-07-04 at 15 28 53
- enable auto-merge functionality (replaces the mergify.yaml`)
    - update the required build names for `main` 
 - remove the `ready-to-merge` label

#### Related issue: 
- https://github.com/kubeshop/botkube/issues/630
- https://github.com/kubeshop/botkube/issues/585

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
 # BotKube
```
Comment (pkosiec) [inline README.md:1]: After a bit of thinking I would vote for replacing `infracloudio` to `kubeshop` in CHANGELOG.md 🙂 For older BotKube versions the old links, license etc. will still stay there on a specific tags, so let's straighten it out for all future releases.

**Review by pkosiec — COMMENTED** [review]
Looks good! A few comments.

File: hack/goreleaser.sh [hunk 2]
```diff
@@ -1,30 +1,12 @@
...
 set -o pipefail
 
 IMAGE_REGISTRY="${IMAGE_REGISTRY:-ghcr.io}"
-IMAGE_REPOSITORY="${IMAGE_REPOSITORY:-infracloudio/botkube}"
-TEST_IMAGE_REPOSITORY="${TEST_IMAGE_REPOSITORY:-infracloudio/botkube-test}"
+IMAGE_REPOSITORY="${IMAGE_REPOSITORY:-kubeshop/botkube}"
+TEST_IMAGE_REPOSITORY="${TEST_IMAGE_REPOSITORY:-kubeshop/botkube-test}"
 IMAGE_SAVE_LOAD_DIR="${IMAGE_SAVE_LOAD_DIR:-/tmp/botkube-images}"
 
 prepare() {
```
[the remark this answers is not in the document]
Comment (mszostok) [reply hack/goreleaser.sh:12]: it's connected with go mod, so it's changed on the https://github.com/kubeshop/botkube/pull/633

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mszostok) [reply README.md:1]: I cannot replace it. The previous release was not repushed, yet.

--


> BTW We can think about get rid of these at some point and support just a single way of installation 🤔 WDYT? To discuss later 🙂

Agree 👍

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -17,62 +17,49 @@ Before you proceed, make sure you have installed BotKube Slack/Mattermost/Teams
...
 
-     1. Edit deploy-all-in-one.yaml and update the configuration.
-        Set SLACK_ENABLED, SLACK_CHANNEL, SLACK_API_TOKEN, clustername, kubectl.enabled and update the resource events configuration you want to receive notifications for in the configmap.
-     2. Create botkube namespace and deploy resources
-     ```sh
-     $ kubectl create ns botkube && kubectl create -f deploy-all-in-one.yaml -n botkube
-     ```
+   ```sh
+   helm install --version v9.99.9-dev --name botkube --namespace botkube --set \<options\> ./helm/botkube
+   ```
```
Comment (pkosiec) [inline CONTRIBUTING.md:62]: Let's remove that 👍

**Review by pkosiec — APPROVED** [review]
LGTM apart from 1 comment

**Pull request merged**